homeassistant/components/overkiz/manifest.json

```diff
@@ -30,5 +30,12 @@
     "pyhumps",
     "pyoverkiz",
     "s3transfer"
-  ]
+  ],
+  "supported_brands": {
+    "cozytouch": "Atlantic Cozytouch",
```

Comment:
This could potentially conflict with https://github.com/Cyr-ius/hass-cozytouch, which uses `cozytouch` domain.

Reply:
https://github.com/home-assistant/core/pull/67015
Custom component are not supported for this feature apparently.

Reply:
Custom components won't support 'supported brands', however they can potentially still conflict. I hope that the initial implementation will be smart enough to handle this.

E.g. when a user has the `hass-cozytouch` integration installed, that it won't do the redirect from cozytouch domain to Overkiz, but than directly goes to the custom component.

Reply:
Will make a note to let custom components overrule supported brand domains.